var/spack/repos/builtin/packages/votca-csg-tutorials/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class VotcaCsgTutorials(CMakePackage):
+    """Versatile Object-oriented Toolkit for Coarse-graining
+       Applications (VOTCA) is a package intended to reduce the amount of
+       routine work when doing systematic coarse-graining of various
+       systems. The core is written in C++.
+
+       This package contains the VOTCA coarse-graining tutorials.
+    """
+    homepage = "http://www.votca.org"
+    url      = "https://github.com/votca/csg-tutorials/tarball/v1.4"
+    git      = "https://github.com/votca/csg-tutorials.git"
+
+    version('develop', branch='master')
```

Comment:
You can call this version `master` now that #1983 has been merged.

Reply:
Meh, I want to keep it consistent with the other votca packages.